src/bundle/Resources/config/services/controllers.yaml

```diff
@@ -77,7 +77,8 @@ services:
         tags:
             - controller.service_arguments
 
-    Ibexa\Bundle\AdminUi\Controller\ContentViewController: ~
+    Ibexa\Bundle\AdminUi\Controller\ContentViewController:
+        parent: Ibexa\Contracts\AdminUi\Controller\Controller
 
     Ibexa\Bundle\AdminUi\Controller\DashboardController: ~
```

Comment:
Same hidden issue here. Not sure if this is the right time to fix it but we probably need to fix it nonetheless.

Reply:
I would say it is, resolved via https://github.com/ibexa/admin-ui/pull/1264/commits/854b37e91ce058d2604f7e27f0742d4ad105f785.